guardian/manage-frontend
# Integrate with discount api

## What does this change?

Integrate with the new discount api to preview and apply discount within digisub saves journey.

Remove the hardcoded price data - instead use the API response for the discounted price and eligibility check. This means the code should work even if prices change.



## How to test

Go to `/digital/?withFeature=digisubSave` and start cancellation journey. See discount offer, click to confirm discount and see next page.



## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
